CONTRIBUTING.md

```diff
@@ -1,50 +1,66 @@
 # Contributing to SlimeVR
 
-This document describes essential knowledge for contributors to SlimeVR.
+This document describes essential knowledge required to contribute to the SlimeVR Server.
 
-## How to get started
+### Prerequisites
 
-### Getting the code
-First, clone the codebase using `git`. If you don't have `git` installed, go install it.
+- [Git](https://git-scm.com/downloads)
+- [Java v17+](https://adoptium.net/temurin/releases/)
+- [Node.js v16+](https://nodejs.org) (We recommend the use of `nvm` instead of installing Node.js directly)
+- [Windows Webview](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section)
```

Comment:
Aside from Windows now including Webview2 by default now, I don't see any mention of the usage of `webkit2gtk` on Linux

Reply:
Can you propose changes. I’m not a Linux nerd.

Reply:
```suggestion
- [Windows Webview](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section)
- [Windows Webview](https://developer.microsoft.com/en-us/microsoft-edge/webview2/#download-section) or ``webkit2gtk`` for Linux
```

Reply:
Is there a link or something that users should know about?

Reply:
not worth it, linux is just kernel, depending on the distro they probably have different ways of installing :p

Reply:
Oki